cmd/openshift-install/create.go

```diff
@@ -440,27 +446,41 @@ func logComplete(directory, consoleURL string) error {
 	if err != nil {
 		return err
 	}
-	logrus.Info("Install complete!")
+	if complete {
+		logrus.Info("Install complete!")
+	}
 	logrus.Infof("Run 'export KUBECONFIG=%s' to manage the cluster with 'oc', the OpenShift CLI.", kubeconfig)
-	logrus.Infof("The cluster is ready when 'oc login -u kubeadmin -p %s' succeeds (wait a few minutes).", pw)
-	logrus.Infof("Access the OpenShift web-console here: %s", consoleURL)
+	if complete {
+		logrus.Infof("The cluster is ready when 'oc login -u kubeadmin -p %s %s' succeeds.", pw, config.Host)
+	} else {
+		logrus.Infof("Depending on the severity of the error, you may be able to log in with: oc login -u kubeadmin -p %s %s", pw, config.Host)
+	}
+	if consoleURL != "" {
+		logrus.Infof("Access the OpenShift web-console here: %s", consoleURL)
+	}
 	logrus.Infof("Login to the console with user: kubeadmin, password: %s", pw)
 	return nil
 }
 
 func finish(ctx context.Context, config *rest.Config, directory string) error {
-	if err := waitForInitializedCluster(ctx, config); err != nil {
-		return err
+	errs := []error{}
+	err := waitForInitializedCluster(ctx, config)
```

Comment:
this is
```
err := <>
if err != nil {
..
}
```
while all other calls are
```
if err = <>; err != nil {
....
}
```

??

Reply:
> ... while all other calls are...

They're all using this `err` ;).  But whatever, I wish `gofmt` would just make this sort of thing consistent with whatever a coin-flip says is the easiest to read :p.